webrtc.html

```diff
@@ -4485,35 +4485,35 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <p>The <code><a href="#widl-RTCPeerConnection-getStats-Promise-RTCStatsReport--MediaStreamTrack-selector">
         getStats()</a></code>
-      method delivers a successful result in the form of a
-      <code><a>RTCStatsReport</a></code> object. A
-      <code><a>RTCStatsReport</a></code> object represents a map between
-      strings, identifying the inspected objects (<a href=
+      method delivers a successful result in the form of an
+      <code><a>RTCStatsReport</a></code> object. An
+      <code><a>RTCStatsReport</a></code> object is a map between
+      strings that identify the inspected objects (<a href=
       "#dom-rtcstats-id">RTCStats.id</a>), and their corresponding
-      <code><a>RTCStats</a></code> objects.</p>
+      <code><a>RTCStats</a></code>-derived dictionaries.</p>
 
       <p>An <code><a>RTCStatsReport</a></code> may be composed of several
-      <code><a>RTCStats</a></code> objects, each reporting stats for one
+      <code><a>RTCStats</a></code>-derived dictionaries, each reporting stats for one
       underlying object that the implementation thinks is relevant for the
       <a href="#stats-selector">selector</a>. One achieves the total for the
       <a href="#stats-selector">selector</a> by summing over all the stats of a
       certain type; for instance, if a <code>MediaStreamTrack</code> is carried
       by multiple SSRCs over the network, the
-      <code><a>RTCStatsReport</a></code> may contain one <code>RTCStats</code>
-      object per SSRC (which can be distinguished by the value of the "ssrc"
+      <code><a>RTCStatsReport</a></code> may contain one <code>RTCStats</code>-derived
+      dictionary per SSRC (which can be distinguished by the value of the "ssrc"
       stats attribute).</p>
 
       <dl class="idl" title="interface RTCStatsReport">
-        <dt>getter RTCStats (DOMString id)</dt>
+        <dt>readonly maplike&lt;DOMString, object&gt;</dt>
 
         <dd>
-          <p>Getter to retrieve the <code><a>RTCStats</a></code> objects that
+          <p>This is a map accessor to retrieve the various dictionaries
```

Comment:
the content of the `&lt;dd>` element is wrapped into a `&lt;p>` element by respec, so this ends up producing invalid markup; while it could be argued that respec is wrong here, can you change this to not use paragraphs?